test_common.py

```diff
@@ -0,0 +1,20 @@
+import pathlib
+import pytest
+import os
+
+import common
+
+
+def test_atomic_write(tmp_path: pathlib.Path) -> None:
```

Comment:
Should we invoke tests somewhere in GH CI?

Reply:
Yes. On `th/host-run` branch (on review), there is a patch to run pytest in github CI.